components/steps/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Steps'
```

Comment:
It's a quite common pattern in multiple programming languages:
> Create a module with a single entry point, so that it defines a public API of the component that other components can use.